test/fail_compilation/imports/diag12598a.d

```diff
@@ -1,3 +0,0 @@
-module imports.diag12598a;
-
-struct lines { }
```

Comment:
You should record into the git history what happened in this PR. Moving the test case to `compiable` would be the best clarify way that the issue 12598 case is now just accepted.
If you don't like it, you should add a comment into the commit message, like "issue 12598 case is now properly accepted, so its test case `fail_compilation/diag12598.d` is deleted.